wildlifelicensing/apps/main/models.py

```diff
@@ -122,7 +122,7 @@ class CommunicationsLogEntry(models.Model):
     documents = models.ManyToManyField(Document, blank=True)
 
     customer = models.ForeignKey(EmailUser, null=True, related_name='customer')
-    officer = models.ForeignKey(EmailUser, null=True, related_name='officer')
+    staff = models.ForeignKey(EmailUser, null=True, related_name='officer')
 
```

Comment:
Changed and checked if anything was using the officer related name, and it wasn't.